RestructuredText/reStructuredText.sublime-syntax

```diff
@@ -77,6 +88,17 @@ contexts:
       scope: markup.underline.substitution.restructuredtext
       captures:
         1: punctuation.definition.substitution.restructuredtext
+    - match: (:math:)(`)
+      comment: links `...`_ or `...`__
```

Comment:
This comment is clearly wrong. Just remove it.

Reply:
Copy paste, whoops

Reply:
```suggestion
      comment: embedded math-mode latex, https://docutils.sourceforge.io/docs/ref/rst/directives.html#math
```